prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -442,6 +445,7 @@ fi_ibv_rdm_process_connect_request(struct rdma_cm_event *event,
 		HASH_ADD(hh, ep->domain->rdm_cm->av_hash, addr,
 			 FI_IBV_RDM_DFLT_ADDRLEN, av_entry);
 	} else {
+		pthread_mutex_lock(&av_entry->conn_lock);
 		HASH_FIND(hh, av_entry->conn_hash, &ep,
 			  sizeof(struct fi_ibv_rdm_ep *), conn);
 		if (!conn) {
```

Comment:
mutex must be released in error path below.

Reply:
Ops, no need to be so quickly :) Done
Thanks!